### ISSUE TITLE
[Bug Report] Output screen problem with LED Panel screen

### ISSUE THREAD

**Issue by Dexiion** [ChurchApps/FreeShow#875]
**Describe the bug**
Last saturday I tried the 1.2.8 version of the app and I had a problem with the output screen, I’m allways getting this two blackbars on the sides of the screen I tried reescaling by my self disabling the Allway on Top function and by pressing Ctrl but I still got the blackbars.

Also when reescaling by hand I press the save button but the next time I open FreeShow It doesn’t save the reescaling that I had.

**Reproducing**
I am not sure on how to reproduce this

**Screenshots (Optional)**

![IMG_1408](https://github.com/user-attachments/assets/6cdebc4e-f604-460a-9435-4b05715feb98)

![IMG_1409](https://github.com/user-attachments/assets/2c9af2e1-b7c5-42f7-ac18-d4ab2d52bfd5)

**Version (Optional)**
 - OS: Windows
 - FreeShow Version: 1.2.8

**Additional context**
Would be nice to have an option to reescale and resize all the output as I want and getting it fitted

**Comment by vassbo** [comment]
First of all update to the latest pre-release, as the manual resize will be properly saved:
https://github.com/ChurchApps/FreeShow/releases/tag/v1.2.9-beta.1

Secondly did you click "Choose screen" in the "Outputs" settings, and click on the output?
Here you can also manually enter pixel values if you would like.

**Comment by Dexiion** [comment]
Hi!

Sorry for reopenning this but I did not figure how to do this.

My led has a resolution of 768x480 I tries to go to the output settings and select the screen, then I get twi black bars at the top and at the bottom of the screen, I tried to set the position and the weight/heigth manually but I have to increase a lot of the width to not have this blackbars and we miss a lot of space of the ouput on the right.

Is there any solution for this?

Thanks!
